pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -133,8 +133,8 @@ func newCertRotationController(
 		certrotation.SigningRotation{
 			Namespace:              operatorclient.OperatorNamespace,
 			Name:                   "aggregator-client-signer",
-			Validity:               30 * rotationDay,
-			Refresh:                15 * rotationDay,
+			Validity:               30 * defaultRotationDay,
```

Comment:
what about other componenets ?

Reply:
like `KubeAPIServerToKubeletClientCert` ?

Reply:
Good question. Have we changed it for other components? This is just a rebase of the same PR we had for the last couple of releases.